internal/character/sampo/skill.go

```diff
@@ -25,9 +26,17 @@ func (c *char) Skill(target key.TargetID, state info.ActionState) {
 }
 
 func (c *char) OnProjectileHit(target key.TargetID, stanceDamage float64, i int) {
-	// if c.info.Eidolon >= 4 && c.engine.HasBehaviorFlag(target, model.BehaviorFlag_STAT_DOT_POISON) {
-	// 	//TODO: implement sampo E4
-	// }
+	if c.info.Eidolon >= 4 && c.engine.HasBehaviorFlag(target, model.BehaviorFlag_STAT_DOT_POISON) {
+		// Wind dot (kit dots)
+		for _, dot := range c.engine.GetModifiers(target, common.WindShear) {
+			dot.State.(common.WindShearState).TriggerDot(dot, 0.06, c.engine, target)
+		}
+
+		// Break wind dots
+		for _, dot := range c.engine.GetModifiers(target, common.BreakWindShear) {
+			dot.State.(common.BreakWindShearState).TriggerDot(dot, 0.06, c.engine, target)
+		}
+	}
```

Comment:
NIT: My FUD with this is you are checking by behavior flag but getting modifiers by name. Runs us into an odd situation for what if there is something that is `STAT_DOT_POISON` but not `WindShear` or `BreakWindShear`. Best if we had some way to `GetModifiersByBehaviorFlag` instead?